src/components/Data/Documents/Page.vue

```diff
@@ -238,6 +267,29 @@ export default {
     }
   },
   methods: {
+    getCoordinates(document) {
```

Comment:
Please, regroup the "geomap" methods with a big comment like the others. It's not very important but helps a little bit to keep this huge component tidy.